src/get-started/java-binding.md

```diff
@@ -28,25 +28,30 @@ Follow these steps:
 
 - Download and unpack the archive from [GitHub releases][github-releases] into
   some known location. Create an environment variable `EXONUM_HOME` pointing at this
-  location. You can execute the following command to add this variable to your `.profile`
-  and preserve its value for every terminal session.
+  location. You can also add an entry to the `PATH` variable.
 
   ```bash
-  echo 'export EXONUM_HOME=/path/to/your/install/location' >> ~/.profile
+  export EXONUM_HOME=/path/to/your/install/location
+  export PATH="$PATH:/path/to/your/install/location"
   ```
 
   This step is not necessary, but will help you in configuring your workflow.
```

Comment:
… is not necessary during installation, but is required to pick/configure the JVM to use by the application: